topics/ui/controls/progress_bar.md

```diff
@@ -21,112 +21,173 @@ Follow the rules for [progress indicators](progress_indicators.md).
+<table style="none" border="false">
+  <tr>
+    <td width="378">
+      <img src="06 process_name.png" alt="Process name above the bar"/>
+    </td>
+    <td>
+      <p>Place a process name above the progress bar.</p>
+    </td>
+  </tr>
+  <tr>
+    <td width="378">
+      <img src="07 process_name_on_the_left.png" alt="Process name on the left"/>
+    </td>
+    <td>
+      <p>If the vertical space is limited, place the process name with a colon on the left.</p>
+    </td>
+  </tr>
+  <tr>
+     <td width="378">
+         <img src="08 header_for_several_processes.png" alt="Header for several processes"/>
+        </td>
+        <td>
+        <p>If there are several processes in a group, add a bold header. The header is a noun.</p>
+     </td>
+  </tr>
+  <tr>
+     <td width="378">
+         <img src="09 inline_processes.png" alt="Inline process"/>
+        </td>
+        <td>
+        <p>Do not show a process name for inline processes.</p>
+     </td>
+  </tr>
+</table>
 
 #### Process details
-
-Place process details under the progress bar in one line:
-
-![](comment.png){width=308}
-
-The length of the comment is limited by the progress bar:
-
-![](comment_long.png){width=308}
-
-If space is limited, show percentage completed:
-
-![](horizontaly.png){width=342}
+<table style="none" border="false">
+  <tr>
+    <td width="378">
+      <img src="10 process_details.png" alt="Process details under the bar"/>
+    </td>
+    <td>
+      <p>Place process details under the progress bar in one line.</p>
+    </td>
+  </tr>
+  <tr>
+    <td width="378">
+      <img src="11 process_details.png" alt="Comment length limited by progress bar."/>
+    </td>
+    <td>
+      <p>The length of the comment is limited by the progress bar.</p>
+    </td>
+  </tr>
+    <tr>
+     <td width="378">
+         <img src="12 process_details.png" alt="Show percentage completed if space is limited"/>
+        </td>
+        <td>
+        <p>If space is limited, show percentage completed.</p>
+     </td>
+    </tr>
+</table>
 
 #### In dialog
 
 Use a process name as a dialog header, capitalize the title and remove ellipsis. Process details appear above the progress bar:
 
-![](dialog.png){width=544}
+![](13 in_dialog.png){width=706}
 
 #### In status bar
 
-Place a process name under the progress bar in the Status bar:
+Place a process name with an ellipsis before the progress bar:
 
-![](status_bar.png){width=238}
+![](14 in_status_bar.png){width=706}
 
 ### Process status
 
 If a process consists of substeps that can fail but do not terminate the process, then use green and red colors to show the intermediate status.
 For example, show the status of the running tests:
 
-![](progress_color.png){width=390}
+![](15 process_status.png){width=706}
 
 Do **not** color progress bar to show the final result of the task, use [notifications](notification_types.md).
 In case of success, show notification for the user-initiated tasks, in case of failure — for all tasks.
```

Comment:
The second sentence is not clear without an example. Possibly remove it as it's not about the progress control. I'd also suggest rewriting the first sentence as:

Do not use red/greed colors to show the final status. After the process is completed, hide the progress bar and show a notification to show the process result.